app-benchmarks/sysbench/sysbench-1.0.19.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools pax-utils
+
+DESCRIPTION="System performance benchmark"
+HOMEPAGE="https://github.com/akopytov/sysbench"
+SRC_URI="https://github.com/akopytov/sysbench/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="aio mysql postgres test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="aio? ( dev-libs/libaio )
+	mysql? ( dev-db/mysql-connector-c:= )
+	postgres? ( dev-db/postgresql:= )
+	dev-lang/luajit:="
```

Comment:
There is not subslot defined, this `:=` seems useless.

Reply:
I thought it means that if the luajit slot changes this will be rebuilt. Same as above.

Reply:
https://devmanual.gentoo.org/general-concepts/dependencies/index.html#slot-dependencies

:= means that any slot is acceptable, and that your package should be rebuilt if the version best matching the runtime dependency is updated to a version with a different slot or subslot;

Reply:
`:=` is used to detect soname changes, like with *mysql-connector-c* the respective sonames for in-tree versions are 
```
/usr/lib64/libmysqlclient.so.21
...
/usr/lib64/libmysqlclient.so.18
```

Between versions 8.0.19 and 6.1.11-r2, which have SUBSLOTS defined for sonames:
**8.0.19 0/21** & **6.1.11-r2 0/18**. The subslot binder will rebuild software **linked** against these sonames to be re-linked again, to prevent breakage. Now, luajit doesn't have a subslot so I doubt it also has breaking soname ABI. 

`:*` can also be used for "any slot to be acceptable", however I don't see luajit slot bump ahead in some time, so why not just put `:2` in there? I really doubt this links to luajit.so for runtime, but granted I didn't check that either.